tests/data-files/examples/0.9.0/item-spec/examples/CBERS_4_MUX_20181029_177_106_L4.json

```diff
@@ -44,6 +44,33 @@
         -4.731829
     ],
     "properties": {
+        "eo:gsd": 20,
+        "eo:bands": [
+            {
+                "name":"B5",
+                "common_name": "blue",
+                "center_wavelength": 0.485,
+                "full_width_half_max": 0.07
+            },
+            {
+                "name":"B6",
+                "common_name": "green",
+                "center_wavelength": 0.555,
+                "full_width_half_max": 0.07
+            },
+            {
+                "name":"B7",
+                "common_name": "red",
+                "center_wavelength": 0.66,
+                "full_width_half_max": 0.06
+            },
+            {
+                "name":"B8",
+                "common_name": "NIR",
+                "center_wavelength": 0.83,
+                "full_width_half_max": 0.12
+            }
+        ],
```

Comment:
i was able to fill in this band information just from googling the CBERS-4 but the same info was not as obvious for DigitalGlobe, Landsat-8 and Planet examples. The tests pass with just an empty list value for `eo:bands` but it may be better if we fill in that info in the example.